src/main/java/com/dev/movietheatre/model/User.java

```diff
@@ -16,6 +18,8 @@
     @Column(name = "email", unique = true, length = 20, nullable = false)
     private String email;
     private String password;
+    @ManyToMany
+    private List<Role> roles;
```

Comment:
better to use Set here, to avoid duplication

Reply:
done